src/TuringTutorials.jl

```diff
@@ -7,117 +7,120 @@ cssfile = joinpath(@__DIR__, "..", "templates", "skeleton_css.css")
 latexfile = joinpath(@__DIR__, "..", "templates", "julia_tex.tpl")
 
 function weave_file(folder,file,build_list=(:script,:html,:pdf,:github,:notebook); kwargs...)
-  tmp = joinpath(repo_directory,"tutorials",folder,file)
-  Pkg.activate(dirname(tmp))
-  Pkg.instantiate()
-  args = Dict{Symbol,String}(:folder=>folder,:file=>file)
-  if :script ∈ build_list
-    println("Building Script")
-    dir = joinpath(repo_directory,"script",folder)
-    isdir(dir) || mkpath(dir)
-    args[:doctype] = "script"
-    tangle(tmp;out_path=dir)
-  end
-  if :html ∈ build_list
-    println("Building HTML")
-    dir = joinpath(repo_directory,"html",folder)
-    isdir(dir) || mkpath(dir)
-    args[:doctype] = "html"
-    weave(tmp,doctype = "md2html",out_path=dir,args=args; fig_ext=".svg", css=cssfile, kwargs...)
-  end
-  if :pdf ∈ build_list
-    println("Building PDF")
-    dir = joinpath(repo_directory,"pdf",folder)
-    isdir(dir) || mkpath(dir)
-    args[:doctype] = "pdf"
-    try
-      weave(tmp,doctype="md2pdf",out_path=dir,args=args; template=latexfile, kwargs...)
-    catch ex
-      @warn "PDF generation failed" exception=(ex, catch_backtrace())
+    tmp = joinpath(repo_directory,"tutorials",folder,file)
+    Pkg.activate(dirname(tmp))
+    Pkg.instantiate()
+    args = Dict{Symbol,String}(:folder=>folder,:file=>file)
+    if :script ∈ build_list
+        println("Building Script")
+        dir = joinpath(repo_directory,"script",folder)
+        isdir(dir) || mkpath(dir)
+        args[:doctype] = "script"
+        tangle(tmp;out_path=dir)
+    end
+    if :html ∈ build_list
+        println("Building HTML")
+        dir = joinpath(repo_directory,"html",folder)
+        isdir(dir) || mkpath(dir)
+        args[:doctype] = "html"
+        weave(tmp,doctype = "md2html",out_path=dir,args=args; fig_ext=".svg", css=cssfile, kwargs...)
+    end
+    if :pdf ∈ build_list
```

Comment:
I'm not sure if we ever want to build PDFs (and in particular add them to the repo). There are discussions over at SciML to remove them.

Reply:
You can, e.g., always print the HTML version to a file or weave it manually.

Reply:
I'd actually lean towards weaving PDFs, just because academic types might prefer to print these out and use the nicer PDF typesetting.

Reply:
We could also just remove it as a default, but keep the functionality.

For starters we're likely going to have to generate everything by hand anyways, simply because the Git Actions, etc. will surely time out before completion.

Reply:
I'm completely fine with keeping the functionality (although just calling `weave` doesn't seem more complicated) but I'm not sure if it's useful to generate them by default and add them to the git repo. Maybe a nice simple functions for generating PDFs could be added (instead), similar to the functionality for opening the notebook/getting the directory below?

Reply:
Cool! I removed PDF from the default.